File: arm.64/langs/lua/src/trip.lua
local type_mod = require("type")

return function()
    return {
        type_mod.Triplet("/version", "{}", "{\"once\":true}"),
        type_mod.Triplet("/storage", "{}", "{\"once\":true}"),
        type_mod.Triplet("sql.help", "{}", "{\"once\":true}")
    }
end

File: arm.64/langs/php/src/trip.php
<?php
require_once __DIR__ . '/type.php';

function trip(): array {
    return [
        new Triplet('/version', '{}', '{"once":true}'),
        new Triplet('/storage', '{}', '{"once":true}'),
        new Triplet('sql.help', '{}', '{"once":true}')
    ];
}
